docs/ansible.eda.project_info_module.rst

```diff
@@ -0,0 +1,228 @@
+
+.. Created with antsibull-docs 2.12.0
+
+ansible.eda.project_info module -- List projects in EDA Controller
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+This module is part of the `ansible.eda collection <https://galaxy.ansible.com/ui/repo/published/ansible/eda/>`_ (version 1.4.7).
```

Comment:
What is this note about? It's unclear to me. Shouldn't this module be part of the 2.0 collection only?

Reply:
This is autgenerated using antsibull-changelog. Need to check how to skip this line in docs for this collection.

Reply:
I will try to update antsibull-docs to address the current issues. I raised bugs and linked them in the description. They should be easy to address.